### DISCUSSION TITLE
Using ARCore/Sceneform for drawing image/texture on floor? (Like "floor is lava")

### DISCUSSION THREAD

**Discussion by PepaZapletal** [sceneview/sceneform-android#229] [Q&A] [answered]
Hi,

is it possible to draw texture/image on the floor with sceneform?

I have a texture for pavement (for example) and I want to go before the house and draw it instead of the current pavement. For a better idea of how it will look. 

We have a web version with works only with images:
![Výstřižek](https://user-images.githubusercontent.com/55383847/145380550-4bbfe6b8-1c9d-4035-9e5c-a597258c7c2e.JPG)

(you can stretch the image of the pavement)

And we want to do something similar for mobile (with using AR). For iOS there is ARKit and with it, it can be maybe possible to do.

Is it something like it for Android? Do you have some suggestions for it?

Thanks for the tips!

**Comment by PepaZapletal** [comment]
It should work similarly to the web app.

The user adds 4 points/vertex in the AR (or more points) and it should draw the texture/image inside these points. If it is possible.

**Comment by PepaZapletal** [comment, marked as answer]
I found this: https://stackoverflow.com/questions/54433707/how-to-draw-a-polygon-with-sceneform-arcore

It's look what like want I want.

**Comment by ekordev** [comment]
> We have a web version with works only with images

Is the web version open source?

**Reply by PepaZapletal** [reply]
No. But here is a link https://www.bestvisio.cz if you want look at it.
